Source/lighting.cpp

```diff
@@ -1213,7 +1213,7 @@ void InitVision()
 
 int AddVision(int x, int y, int r, BOOL mine)
 {
-	int vid; // BUGFIX: if numvision >= MAXVISION behavior is undefined
+	int vid = -1;

```

Comment:
```suggestion
 	int vid -1; // BUGFIX: if numvision >= MAXVISION behavior is undefined (fixed)
```
Just to make sure i don't revert it during some merge conflict :)

Reply:
u fked up, where's - :D